### ISSUE TITLE
Accessibility via AccessKit

### ISSUE THREAD

**Issue by sportfloh** [maycoon-ui/maycoon#48]
**Is your feature request related to a problem? Please describe.**
maycoon is not at all accessible for screan readers (for example VoiceOver on macOS). It should be implemented early on to deal with potencial problems and challenges.

**Describe the solution you'd like**
Use [AccessKit](https://crates.io/crates/accesskit) to support accessibility (for example screen readers).
